test/acceptance/to-js-test.js

```diff
@@ -81,4 +81,27 @@ describe('acceptance - to-js', function() {
       }).catch(done);
     });
   });
+  it('handles option --dotted', function(done) {
+    let ps = spawn(process.execPath, [
+      'bin/ember-i18n-csv.js',
+      'to-js',
+      '--csv-path=test/fixtures/i18n-dotted.csv',
+      '--locales-path=tmp/locales',
+      '--merge'
```

Comment:
I've fixed this but test fails still remain. I've rewritten these lines after first revert, that's why I missed some parts.
Is there any debug options for areDirsEqual?